pam/internal/adapter/model.go

```diff
@@ -33,14 +34,14 @@ type sessionInfo struct {
 	encryptionKey string
 }
 
-// model is the global models orchestrator.
-type model struct {
-	pamMTx pam.ModuleTransaction
-	client authd.PAMClient
+// UIModel is the global models orchestrator.
+type UIModel struct {
+	PamMTx              pam.ModuleTransaction
+	Client              authd.PAMClient
+	InteractiveTerminal bool
```

Comment:
Yeah, sorry I was about to push it, then I decided to move them into a different structure, since in gdm I may need to pass another field around, and so I think it's better to manage it via another structure (se latest commit).